api/tests/test_views.py

```diff
@@ -1,64 +1,153 @@
 import pytest
+import json
 
+from django.test import TestCase
+from django.utils import timezone
+from rest_framework.test import RequestsClient
 
-def test_fire():
-    """
-    Employees related test
-    """
-    pass
+from ..factories import EmployeeFactory, IncomeFactory, FutureEventFactory, TodaysEventsFactory, PastEventFactory
 
+SERVER_ADDRESS = 'http://127.0.0.1:8000'
+CLIENT = RequestsClient()
+EMPLOYEES = 'employees'
+EVENTS = 'events'
+INCOMES = 'income'
 
-def test_hire():
-    """
-    Employees related test
-    """
-    pass
 
+def _url_not_detail(content, method='', options=''):
+    return SERVER_ADDRESS + '/api/{}/{}/{}'.format(content, method, options)
```

Comment:
AFAIK when using django test client there's no need to specify the server address. Also we're in Python 3.7 so we use f-strings here:

`return f"/api/{content}/{method}/{options}"`

Reply:
done